Tests/RouterTests.swift

```diff
@@ -77,7 +88,95 @@ class RouterTests: QuickSpec {
             RouterTestServer.disable()
             Router.disableAll()
         }
-        
+
+        describe("Describe Cancelling requests") {
+            var router: Router!
+            let baseURL = "http://www.funky-cancel-request.com"
+            let latency = NSTimeInterval(2)
+
+            beforeEach {
+                router = Router.register(baseURL)
+                router.latency = latency // very high latency to allow us to stop the request before execution
```

Comment:
Since the request is always asynchronous theoretically also 0 latency should work